x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_endpoint/endpoint_info.tsx

```diff
@@ -38,13 +39,24 @@ export const EndpointModelInfo: React.FC<EndpointInfoProps> = ({ endpoint }) =>
       ? serviceSettings.model
       : undefined;
 
+  const { euiTheme } = useEuiTheme();
+
   return (
     <EuiFlexGroup gutterSize="s" alignItems="center">
```

Comment:
Let's add `wrap` to this flex group.  With the new badge, it's more likely this will need to break to two lines.

Reply:
Updated